src/utils/network/session-helper.js

```diff
@@ -91,7 +91,7 @@ export const placeFileInContainer = async (session, fileObject, containerUrl) =>
  */
 
 export const getPodUrl = (username) => {
-  const podOidcIssuer = localStorage.getItem('oidcIssuer');
+  const podOidcIssuer = ENV.VITE_SOLID_IDENTITY_PROVIDER;
```

Comment:
Temp fix so set permission forms would not break for now.

Reply:
I don't think the permission forms are relevant to this PR. If there's an issue with them, you can fix them in a separate PR. Either way, this isn't a good fix. It ties the app to the ID provider set in the ENV, not the one used on login. That will create more bugs in the system than it fixes.

I wouldn't get too attached to those forms anyway. They're really just demo forms, and they need to be rewritten.

Reply:
It's just temporary. The function will get dropped soon once we moved over to using podUrls. For now, I'm going to keep this one in this PR.

Reply:
I think you'd be surprised how long temporary fixes last. If you're determined to keep these forms working, I'd prefer if you put in a lasting fix. This looks like something that can be easily forgotten about until it causes nasty bugs later.

Reply:
I'll make sure to keep a note of this. It'll be important for when we work on Issue #307.

Reply:
Let's revert this. This function has dependencies outside of just the permissions forms, and it's too difficult for reviewers to test the full effects of this change. I don't think we have any designs where the permission forms are in the final version of the app anyway, so if they're broken it's not a major issue.

It's also not clear to me what the issue you're trying to fix here is. I'd like to be able to look over it in another PR.

Reply:
The issue here is we're no longer using localStorage for oidcIssuer. We're now storing the provider in the environment variable.

This function is mainly used for getting the podUrl from a username which requires the provider. As mentioned, it's just changed so that it wouldn't be broken for the mean time. When we start updating the form to use podUrl directly, this function will be removed.

By the time we transition to use podUrl implicitly based on what we plan for in Figma instead of the form, we don't need to mind this issue at all. Which is why I think it's fine as is.

Reply:
I'm not sure what you mean by the first sentence. `oidcIssuer` is still in local storage. the env is just used for initialization.

Everything else in the PR looks good for now, but I'm still pretty confident this change causes a regression. Revert this and I'll approve. We can take a look at this again in a separate PR.

It's best practice to keep PRs focused on one thing. It makes it easier to test and review. A large feature PR like this is already pretty demanding. I don't want to delay the rest of this PR to review this largely tangential change.

Reply:
I didn't broke before the recent changes in Development, so I just wanted to patch it up since it's now part of the profile page.

I've found the `setItem` call in the `OidcLoginComponent` Line 17, but it doesn't seem to be setting the `oidcIssuer` correctly after login (see screenshot). Because of that, it's crashing the set permission form, so I thought wasn't used for `oidcIssuer`, hence the motivation behind the temporary fix for `getPodUrl`.

![Screen Shot 2023-07-25 at 21 23 28 (2)](https://github.com/codeforpdx/PASS/assets/14917816/ec2219d1-127f-4041-b9df-a3c15823da8b)

Think I got this fixed now by moving the setItem outside of `loginHandler` in `OidcLoginComponent`, so I could revert this back.

![Screen Shot 2023-07-25 at 21 33 32 (2)](https://github.com/codeforpdx/PASS/assets/14917816/45624f01-feda-49ee-9f4e-b76f4503d48c)

Reply:
Going to move this into a separate PR.